packages/main-menu-react/package.json

```diff
@@ -0,0 +1,59 @@
+{
+    "name": "@fremtind/jkl-main-menu-react",
+    "version": "0.0.1",
+    "publishConfig": {
+        "access": "public"
+    },
+    "description": "Hovedmenykomponent for Jøkul-løsninger",
+    "homepage": "https://jokul.fremtind.no",
+    "keywords": [
+        "Menu",
+        "Responsive",
+        "Jøkul",
+        "Fremtind"
+    ],
+    "license": "MIT",
+    "types": "./build/index.d.ts",
+    "main": "./build/cjs/index.js",
+    "module": "./build/esm/index.js",
+    "browser": "./build/browser/index.js",
+    "directories": {
+        "lib": "build"
+    },
+    "files": [
+        "build"
+    ],
+    "scripts": {
+        "clean": "rimraf node_modules/ .cache/ build/ dist/ public/ **/*.css",
+        "build:types": "tsc -p tsconfig-for-declarations.json",
+        "build:scripts": "rollup --config ../../rollup.config.js",
+        "build": "run-s build:*",
+        "test": "echo \"Error: run tests from root\" && exit 1",
+        "dev:style": "lerna exec --scope=@fremtind/jkl-logo yarn build:watch",
+        "dev:js": "parcel documentation/index.html",
+        "dev": "run-p dev:*"
+    },
+    "dependencies": {
+        "@babel/runtime": "^7.9.0",
+        "classnames": "^2.2.6",
+        "@fremtind/jkl-hamburger-react": "^1.4.4",
+        "@fremtind/jkl-react-hooks": "^1.6.11",
+        "framer-motion": "^2.6.15"
```

Comment:
Burde Framer være en peerDependency? Selvom det er lite for et animasjonsbibliotek, så er det fortsatt relativt stort, så man ønsker ikke å ha 8 versjoner av den durende rundt.

Reply:
Det er et godt poeng. Den bør i hvert fall være det om vi skal tilby en versjon med ren CSS-animasjon